tests/soak-tests/src/test/java/org/apache/activemq/artemis/tests/soak/brokerConnection/mirror/ReplicatedBothNodesMirrorTest.java

```diff
@@ -213,15 +221,17 @@ private static void replaceLogs(File serverLocation) throws Exception {
                                       "logger.artemis_utils.level=INFO\n" + "\n" +
                                          "logger.endpoint.name=org.apache.activemq.artemis.core.replication.ReplicationEndpoint\n" +
                                          "logger.endpoint.level=INFO\n" +
+                                         "logger.ack.name=org.apache.activemq.artemis.protocol.amqp.connect.mirror.AckManager\n" +
+                                         "logger.ack.level=TRACE\n" +
```

Comment:
Running a soak with trace logs seems a bit weird...intended?

Reply:
Yep, I didnt mean it isnt something you would deliberately do when e.g dev/debugging, just that for the default execution of the test it would seem unusual. Especially since 'logging change alters behaviour' is a thing as ou noted; so we should test the more expected setup in the default case to cover what people are more likely to be doing.

Reply:
I did... and I'm ready to make this false now... I had a test issue...


the new PRs should have this false now.